src/plugins/candyMachineModule/createCandyMachine.ts

```diff
@@ -23,11 +23,7 @@ export type CreateCandyMachineOperation = Operation<
   CreateCandyMachineOutput
 >;
 
-export type CreateCandyMachineInput = {
-  // Data.
-  // TODO(loris): spread data into the input directly.
-  candyMachineData: CandyMachineData;
-
+export type CreateCandyMachineInput = CandyMachineData & {
```

Comment:
This is nice but it prevents us from making some fields optional such as the `uuid` that can be inferred by the handler if not provided.

Reply:
Yeah that is planned, just please let me get there once I have a use case for it first.
I hope to run into one when I'm working on the manager.

Reply:
For instance when managing a Candy Machine all data is required (existing values + updated values), but a missing UUID should _not_ be filled in at that point as the user most likely made an error when not filling it in.

So I'm not ready to flesh out the details here and therefore came up with the simplest solution for now.